t/04-scheduler.t

```diff
@@ -303,7 +303,6 @@ subtest 'job grab (WORKER_CLASS mismatch)' => sub {
     $worker_db_obj->discard_changes;
     is(undef, $sent->{$worker->{id}}->{job}, 'job not grabbed due to default WORKER_CLASS');
     is_deeply($allocated, [], 'no workers/jobs allocated');
-    is($worker_db_obj->t_seen, $last_seen, 't_seen has not changed');
```

Comment:
The `$last_seen` variable is now likely unused.

Reply:
It's still used in the line `cmp_ok($worker_db_obj->t_seen, '>', $last_seen, 'last seen timestamp of worker updated on registration');`